templates/layouts/grid-image.php

```diff
@@ -30,6 +30,10 @@
 
 			$count = 1;
 			$food_term = get_term_by( 'id', $food_id, 'food_menu_cat' );
+			if( !isset( $food_term ) || empty( $food_term ) ) {
```

Comment:
I am confused why isset and empty check. Can't any one of there will do the same task. Also coding standard.

Reply:
See https://codex.wordpress.org/Function_Reference/get_term_by and know what the returned value is and follow the procedural check.

Reply:
okay. I thought `$food_term` can be 0 which is empty but set. `empty()` alone should do the task.